lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,35 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { tabData } from '../../data';
+
+ const Tab = props => {
+      
+      // if(props.tab === props.selectedTab){
+      //   this.setState({ tab: tabData})
+      // } else {
+      //   this.setState({ tab: tabData })
+      // }
 
-const Tab = props => {
-  /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
-      if they match, the className should be: 'tab active-tab', 
-      if it is not it should just be 'tab'*/
   return (
     <div
-      className={''}
+      className={props.tab === props.selectedTab ? "tab active-tab" : 'tab'}
```

Comment:
Nice use of a ternary here top set your className, you could also use a template string like in our groceries list className={`tab ${props.tab === props.selectedTab ? " active-tab" : ""}}